packages/amplication-server/src/core/resource/resourceBtm.service.ts

```diff
@@ -360,7 +359,7 @@ export class ResourceBtmService {
         })),
       };
     } catch (error) {
-      throw new GptBadFormatResponseError(JSON.stringify(promptResult), error);
+      throw new GptBadFormatResponseError(promptResult);
```

Comment:
Please add the original `error` that was removed so we get it logged and don't lose information